doc/faq.rst

```diff
@@ -26,6 +26,27 @@ See :ref:`contributing`. Before wanting to add a new algorithm, which is
 usually a major and lengthy undertaking, it is recommended to start with :ref:`known
 issues <easy_issues>`.
 
+What's the best way to ask questions about scikit-learn?
```

Comment:
Yours is better than mine, but it kind of removes the bug report component. People are also unsure about where to report bugs or ask if something is a bug.